src/main/scala/cromwell/backend/provider/local/LocalBackendActor.scala

```diff
@@ -88,9 +89,9 @@ class LocalBackendActor(task: TaskDescriptor) extends BackendActor with StrictLo
       val command = initiateCommand()
       logger.debug(s"Creating bash script for executing command: $command.")
       writeBashScript(command, executionDir)
-      notifyToSubscribers(new TaskStatus(Status.Created))
+      notifySubscribers(CreatedTaskStatus(stdout.toString, stderr.toString))
```

Comment:
First q should be answered above.  These are just pointers, the call has likely not even started executing yet.